docs/windows.md

```diff
@@ -21,3 +21,20 @@ local kp = (import 'kube-prometheus/main.libsonnet') +
 ```
 
 [Containerd](https://github.com/prometheus-community/windows_exporter/blob/master/kubernetes/kubernetes.md) version can run as pod.
+
+```
+local kp = (import 'kube-prometheus/main.libsonnet') +
```

Comment:
Can we update the docs above that say `Currently, Docker based Windows does not support running with [windows_exporter](https://github.com/prometheus-community/windows_exporter) in a pod so this add on uses [additional scrape configuration](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/additional-scrape-config.md) to set up a static config to scrape the node ports where windows_exporter is configured.`?

I would also lean towards moving this up in the docs and making it the default over the static pod configuration.  We should drop the static pod configuration in the future.

Reply:
Sure, I have updated the doc to reflect both configs and moved hostprocess config up.